srcts/types/extras/globalShiny.d.ts

```diff
@@ -1,8 +1,7 @@
-import type { Shiny as RStudioShiny } from "../src/shiny/index";
+import type { Shiny as ShinyClass } from "../src/shiny/index";
 declare global {
-    const Shiny: RStudioShiny;
+    const Shiny: ShinyClass;
     interface Window {
-        Shiny: RStudioShiny;
+        Shiny: ShinyClass;
     }
-    type Shiny = RStudioShiny;
```

Comment:
The removal of this type leads to some required changes in bslib.
Note that the change is only with types; the JS code does not need to change.
https://github.com/rstudio/bslib/pull/1054